cts/scheduler/scores/group-dependents.scores

```diff
@@ -1,6 +1,6 @@
 
 drbd:0 promotion score on asttest1: 1
-drbd:1 promotion score on asttest2: 9
+drbd:1 promotion score on asttest2: 7
```

Comment:
From commit message: "...the colocation chain will now take into account all of ip_voip_addresses_p's members rather than just the group itself."

How did we lose two points for `drbd` on node 2 after taking into account all members? There are no negative scores. Did we previously count all members plus the two active members, and now we count only "all members"? Likewise we lost one point for `ms_drbd`. Just want to make sure our math is still how we want it.

At this point I have not built this PR and tried to trace it; you might know off-hand.

Reply:
Maybe some kind of double-counting in the current version. Running a quick trace on 2.1 (and acknowledging that it goes from drbd to ms_drbd, but the scores seem to track):
```
(pcmk__native_merge_weights)    trace: drbd:1: Merging scores from mysqld (at 1.000000)
(pcmk__add_scores)      trace: 1 + 0 = 1
(add_node_scores_matching_attr)         trace: asttest2: 0 + 1.000000 * 1 = 1
...
(pcmk__native_merge_weights)    trace: drbd:1: Merging scores from mysqld (at 1.000000)
(pcmk__add_scores)      trace: 1 + 1 = 2
(add_node_scores_matching_attr)         trace: asttest2: 1 + 1.000000 * 1 = 2
...
(add_sort_index_to_node_weight)         trace: Adding sort index 5 of drbd:1 to weight for asttest2
(pcmk__add_scores)      trace: 5 + 2 = 7
...
(pcmk__native_merge_weights)    trace: ms_drbd: Merging scores from mysqld (at 1.000000)
(pcmk__add_scores)      trace: 1 + 7 = 8
(add_node_scores_matching_attr)         trace: asttest2: 7 + 1.000000 * 1 = 8
...
(pcmk__native_merge_weights)    trace: ms_drbd: Merging scores from mysqld (at 1.000000)
(pcmk__add_scores)      trace: 1 + 8 = 9
(add_node_scores_matching_attr)         trace: asttest2: 8 + 1.000000 * 1 = 9
```

Reply:
I misunderstood the impact when I first made this change.

Looking at it again, the cmp_resources() logic inside pcmk__add_colocated_node_scores() should be in pcmk__group_add_colocated_node_scores() instead, then the last two direct calls to pcmk__add_colocated_node_scores() can be replaced with the method call, and the group-specific logic can be removed from pcmk__add_colocated_node_scores().

However there are other differences in the primitive vs group calculation that I need to look at more carefully.

Reply:
I'm dropping all the commits related to replacing direct calls with method calls. That should be a separate PR since it has bigger impact.